tests/integration/test_fim/test_synchronization/test_sync_time.py

```diff
@@ -0,0 +1,162 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: Check when the 'wazuh-syscheckd' daemon is performing a synchronization, a normal synchronization will end
+before the configured `interval` and `max_interval`.
+
+components:
+    - fim
+
+suite: synchronization
+
+targets:
+    - agent
+    - manager
+
+daemons:
+    - wazuh-syscheckd
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
+    - Windows Server 2019
+
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/capabilities/file-integrity/index.html
+    - https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/syscheck.html#synchronization
+
+pytest_args:
+    - tier:
+        0: Only level 0 tests are performed, they check basic functionalities and are quick to perform.
+        1: Only level 1 tests are performed, they check functionalities of medium complexity.
+        2: Only level 2 tests are performed, they check advanced functionalities and are slow to perform.
+
+tags:
+    - fim_synchronization
+'''
+import os
+import pytest
+
+
+from wazuh_testing import global_parameters
+from wazuh_testing.tools import LOG_FILE_PATH, configuration
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.modules import TIER2, AGENT, SERVER
+from wazuh_testing.modules import fim
+from wazuh_testing.modules.fim import event_monitor as evm
+
+# Marks
+pytestmark = [AGENT, SERVER, TIER2]
```

Comment:
When you add the `AGENT` tag, Does it know on what agent is run?

Reply:
No it is to make it run in agents. If I want to limit it to Linux agents, it would need also de Linux tag.